text/0000-add-status-field.md

```diff
@@ -0,0 +1,86 @@
+# Meta
+[meta]: #meta
+- Name: Add Status to RFC Metadata
+- Start Date: 2021-07-14
+- Author(s): @aemengo, @ekcasey
+- Status: Draft <!-- Acceptable values: Approved, On Hold, Draft -->
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: (put "N/A" unless this replaces an existing RFC, then link to that RFC)
+
+# Summary
+[summary]: #summary
+
+This is a proposal to add a `Status:` field to the `Meta` section of the RFC template.
+
+# Definitions
+[definitions]: #definitions
+
+**Core team:** The party that votes on the approval of RFCs.
+
+# Motivation
+[motivation]: #motivation
+
+This RFC formally introduces the **"On Hold"** status for an RFC, in a way that's clear to the community.
+
+# What it is
+[what-it-is]: #what-it-is
+
+A new `Status:` field in the `Meta` section of the RFC template. See [#meta](#meta) for example.
+
+The acceptable values are distinguished in the following ways:
+- **Draft**: The Core team has not yet approved the RFC. Edits are still being made.
+- **Approved**: The Core team has approved the RFC and is accepting PRs for its implementation.
+- **On Hold**: The Core team has approved the RFC, but PRs are currently not being accepted for its implementation. A reason must be provided in the RFC.
+
+The `Status:` field does not intend to reflect states that are already implied by the RFC process. Examples including: *In Review*, *Closed*, etc.
+
+# How it Works
+[how-it-works]: #how-it-works
+
+We will add the following to the RFC `Meta` section template:
+
+```
+- Status: Draft <!-- Acceptable values: Approved, On Hold, Draft -->
+```
+
+When an RFC is approved, the [../merge-rfc.sh](../merge-rfc.sh) script reflects the change like so:
+
+```
+- Status: Approved
+```
+
+When an RFC is put on hold (_via Pull Request_) then change is reflected like so:
+
+```
+- Status: On Hold
+```
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+* This field might get out of sync if not properly maintained.
+
+# Alternatives
+[alternatives]: #alternatives
+
+- Communicate "On Hold" status through word of mouth
+- Reject the "On Hold" status altogether
+
+# Prior Art
+[prior-art]: #prior-art
+
+- TensorFlow has an ["Status" field](https://github.com/tensorflow/community/blob/master/rfcs/yyyymmdd-rfc-template.md)
+
+# Unresolved Questions
+[unresolved-questions]: #unresolved-questions
+
+- Should **"Implemented"** be one of the accepted values?
+    - Perhaps, but then what's the process of keeping the _Implemented_ status in sync?
```

Comment:
What's the goal we're trying to solve with the "Status" field? I think answering this would help answer this question.

Reply:
Based on the motivation of this RFC, it seems as though it's to clearly portray the status of any given RFC. If that's the case I would argue that knowing whether something is "Implemented" or "Implementable" is a helpful distinction.

As a prior art: Tekton TEPs use these statuses: https://github.com/tektoncd/community/tree/main/teps#teps